tests/backup/backup_helper.go

```diff
@@ -1223,11 +1404,64 @@ func restoreSuccessCheck(restoreName string, orgID string, retryDuration int, re
 
 	resp, err := backupDriver.InspectRestore(ctx, restoreInspectRequest)
 	if err != nil {
-		return false, err
+		return false, nil, err
 	}
 	restoreStatus := (resp.GetRestore().GetStatus().Status == api.RestoreInfo_StatusInfo_PartialSuccess) || (resp.GetRestore().GetStatus().Status == api.RestoreInfo_StatusInfo_Success)
 	log.Infof("[%s] restored successfully", restoreName)
-	return restoreStatus, nil
+	return restoreStatus, resp, nil
+}
+
+// returns the restoreContext
+func ValidateRestore(restoreInspectResponse *api.RestoreInspectResponse, backupContext *BackupRestoreContext, namespaceMapping map[string]string) (*BackupRestoreContext, error) {
```

Comment:
This should be named `getRestoreContext`

Reply:
I see how that would seem to make sense, but in a future PR, more code will be added to this, which will do "deeper validation", which actually involves the exact code which is shown here i.e. Depper validation inherently creates a `restoreContext`, which I'm simply returning in this case. Other functions which use `CreateRestore` in the future will also use `ValidateRestore` and they'll most likely also have to use the returned context (which was generated as part of validation), as the returned `restoreContext` can be used as part of `ValidateAndDestroy` (if one wants to) as sometimes, it makes sense to clean up the namespace which were restored on the destination.

With all that in mind, this function, the way it right now, makes sense.